### PR TITLE
Update 40-plex-first-run

### PR DESCRIPTION
**Pull request by mxmeeple**
Allow the use of PLEX_CLAIM_FILE for docker swarm purposes

(I tried to test but my knowledge isnt great, so its more of a suggestion)

### CODE REVIEW

**Comment by saltydk** [comment]
Docker Swarm can use ENVs.

**Comment by mxmeeple** [comment]
Sorry I meant secrets not ENVs

**Comment by mxmeeple** [comment]
Sorry I meant secrets not ENVs

File: root/etc/cont-init.d/40-plex-first-run [hunk 1]
```diff
@@ -100,6 +100,8 @@ fi
 
 # Get server token and only turn claim token into server token if we have former but not latter.
 token="$(getPref "PlexOnlineToken")"
+if [ -z "${PLEX_CLAIM}" ] && [ ! -z "${PLEX_CLAIM_FILE}" ]; then
```
Comment (gbooker) [inline root/etc/cont-init.d/40-plex-first-run:103]: The second test here is likely better as
```suggestion
if [ -z "${PLEX_CLAIM}" ] && [ -f "${PLEX_CLAIM_FILE}" ]; then
```

File: root/etc/cont-init.d/40-plex-first-run [hunk 2]
```diff
@@ -100,6 +100,8 @@ fi
 
 # Get server token and only turn claim token into server token if we have former but not latter.
 token="$(getPref "PlexOnlineToken")"
+if [ -z "${PLEX_CLAIM}" ] && [ -f "${PLEX_CLAIM_FILE}" ]; then
+  PLEX_CLAIM=$(cat ${PLEX_CLAIM_FILE})
```
Comment (gbooker) [inline root/etc/cont-init.d/40-plex-first-run:104]: Oh wait, this is missing the `fi`

**Pull request merged**